.rubocop.yml

```diff
@@ -7,3 +7,7 @@ AllCops:
 
 Style/ConditionalAssignment:
   Enabled: false
+
+Style/SymbolArray:
+  Exclude:
+    - lib/expansion_rules.rb
```

Comment:
I'd have gone for the comment in the file, as I think that is a little easier to understand compared to this, but I don't have a strong preference.